app.py

```diff
@@ -40,9 +35,20 @@
 
 # Mastodon
 try:
+    client_file = param['mastodon']['client_id_file']
```

Comment:
Added some sanity checks here because I had a typo in my config file that was tricky to debug.